nuxt.config.ts

```diff
@@ -111,7 +111,15 @@ const config: NuxtConfig = {
   modules: [
     '@nuxt/content',
     '@nuxtjs/axios',
-    'nuxt-lazy-load'
+    'nuxt-lazy-load',
+    '@nuxtjs/redirect-module'
```

Comment:
That's strange, it's working on my machine. I don't know about external links, but there are at least some Slack comments that link to it. If it's not too much hassle, I'd prefer to redirect as it's not super obvious how to get to the new page if the old one's broken.

Reply:
The Redirect Module apparently doesn't work well in Nuxt's static mode without a server (https://github.com/nuxt-community/redirect-module#gotchas).

Reply:
What should I do then? I'd like to close this soon

Reply:
Let me quickly try and find the analytics info about this URL and I'll get back to you asap.

Reply:
The old path is apparently linked from a couple of external resources, so it would be a good idea to setup a redirection indeed.

This should be done preferably done without the "Redirect Module", which doesn't work for our static builds.

Reply:
Sorry it took me a while to get round to it, should be done now